customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+    out.poll_removeUserTitle = "Usuń kolumnę";
+    out.poll_editOption = "Jesteś pewien, że chcesz edytować tę opcję?";
+    out.poll_editOptionTitle = "Edytuj wiersz";
+    out.poll_editUser = "Jesteś pewien, że chcesz edytować tego uzytkownika?";
+    out.poll_editUserTitle = "Edytuj kolumnę";
+
+    out.poll_titleHint = "Tytuł";
+    out.poll_descriptionHint = "Opis";
+
+    // index.html
+
+    out.main_p1 = 'CryptPad to edytor czasu rzeczywistego stawiający na wspólpracę i <strong>prostotę użytkowania</strong>. Szyfrowanie w twojej przeglądarce, chroni twoje dane przed serwerem, chmurą i ABW. Tajny klucz jest przechowywany w adresie strony <a href="https://pl.wikipedia.org/wiki/Uniform_Resource_Identifier#Pe.C5.82ny_wz.C3.B3r_URI_hierarchicznych.5B2.5D">identyfikator fragmentu w URI</a> którego nigdy nie przekazujemy serwerowi, a dostępny jest dla javascript. Udostępniając URL(link), przekazujesz innym upoważnienie do uczestnictwa.';
+    out.main_p2 = 'Ten projekt wykorzystuje wizualny edytor <a href="http://ckeditor.com/">CKEditor</a> , <a href="https://codemirror.net/">CodeMirror</a>, oraz silnik czasu rzeczywistego <a href="https://github.com/xwiki-contrib/chainpad">ChainPad</a>.';
+    out.main_howitworks = 'Jak to działa';
+    out.main_howitworks_p1 = 'CryptPad wykorzystuje wariant algorytmu <a href="https://en.wikipedia.org/wiki/Operational_transformation">Transformacji operacyjnej</a> który jest wstanie odnaleźć rozdzielony konsensus wprowadzanych danych. Używa do tego <a href="https://bitcoin.org/bitcoin.pdf">Łańcuch blokowy Nakamoto</a>, twór zpopularyzowany przez <a href="https://en.wikipedia.org/wiki/Bitcoin">Bitcoin</a>. W ten sposób algorytm może pominąć potrzebę centralnego serwera do rozwiązywania Konfliktów Operacji Przekształcania poprzez Edycję. Bez potrzeby rozwiązywania konfliktów, serwer może pozostać w niewiedzy o zawartości która jest edytowana w dokumencie.';
+    out.main_about = 'O projekcie';
+    out.main_about_p1 = 'Możesz przeczytać więcej o naszej <a href="/privacy.html" title="">polityce prywatności</a> oraz <a href="/terms.html">warunkach korzystania z usług</a>.';
+
+    out.main_about_p2 = 'Jeżeli masz jakieś pytania lub komentarze, możesz napisać na <a href="https://twitter.com/cryptpad">tweeterze</a>, otworzyć problem na <a href="https://github.com/xwiki-labs/cryptpad/issues/" title="our issue tracker">githubie</a>, przywitać się na ircu (<a href="http://webchat.freenode.net?channels=%23cryptpad&uio=MT1mYWxzZSY5PXRydWUmMTE9Mjg3JjE1PXRydWUe7" title="freenode webchat">irc.freenode.net</a>), lub wysłać nam <a href="mailto:research@xwiki.com">email</a>.';
+
+    out.table_type = 'Typ';
+    out.table_link = 'Link';
+    out.table_created = 'Stworzone';
+    out.table_last = 'Ostatnio otwarte';
+
+    out.button_newpad = 'STWÓRZ PAD WYSIWYG';
+    out.button_newcode = 'STWÓRZ PAD DO KODU';
+    out.button_newpoll = 'STWÓRZ GŁOSOWANIE';
```

Comment:
To discuss.
Well. Because what you do with that poll is vote :D
Correct translation would be "Stwórz Balot", but it's a word that isn't used in common language and may result in misunderstandings.

Though I've sticked with "Balot" in the list as it's the only short word that fits.